requirements.txt

```diff
@@ -9,6 +9,6 @@ semver
 requests-cache
 rich==13.3.4
 
-multiversx-sdk-core>=0.6.0,<0.7.0
-multiversx-sdk-network-providers>=0.11.0,<0.12.0
-multiversx-sdk-wallet>=0.8.0,<0.9.0
+multiversx-sdk-core==0.7.1
```

Comment:
It's good to use fixed versions, since `mxpy` is an end-application (not a library) :+1: 

Plus the fact that it's always installed within a venv (at least, this is what mxpy-up does, this is what we recommend).

Reply:
On the other hand, it's also nice to automatically benefit from hotfixes in dependencies (thus `>=` with `<` would have also worked).

Reply:
I'm a big fan of fixed versions, but I agree it's nice to benefit from hotfixes. Will indeed change to use `<`.